Update docs for `Animation.track_set_path` and `AnimationMixer.root_motion_track`

scotmcp opened the pull request:
Updated the usage description of [method Animation.track_set_path] and [member Animation.root_motion_track] to identify the parent is the root_node identified in animation player, and that the format is the same for each.

AThousandShips commented on doc/classes/AnimationMixer.xml:
```diff
@@ -304,8 +304,8 @@
-			The path to the Animation track used for root motion. Paths must be valid scene-tree paths to a node, and must be specified starting from the parent node of the node that will reproduce the animation. To specify a track that controls properties or bones, append its name after the path, separated by [code]":"[/code]. For example, [code]"character/skeleton:ankle"[/code] or [code]"character/mesh: ...
-			If the track has type [constant Animation.TYPE_POSITION_3D], [constant Animation.TYPE_ROTATION_3D] or [constant Animation.TYPE_SCALE_3D] the transformation will be canceled visually, and the animation will appear to stay in place. See also [method get_root_motion_position], [method get_root_motion_rotation], [method get_root_motion_scale] and [RootMotionView].
+			The path to the Animation track used for root motion. Paths must be valid scene-tree paths to a node, and must be specified starting from the parent node of the node that will reproduce the animation. The [member root_motion_track] uses the same format as [method Animation.track_set_path], but note that a bone must be specified.
++			If the track has type [constant Animation.TYPE_POSITION_3D], [constant Animation.TYPE_ROTATION_3D], or [constant Animation.TYPE_SCALE_3D] the transformation will be canceled visually, and the animation will appear to stay in place. See also [method get_root_motion_position], [method get_root_motion_rotation], [method get_root_motion_scale], and [RootMotionView].
```
```suggestion
			If the track has type [constant Animation.TYPE_POSITION_3D], [constant Animation.TYPE_ROTATION_3D], or [constant Animation.TYPE_SCALE_3D] the transformation will be canceled visually, and the animation will appear to stay in place. See also [method get_root_motion_position], [method get_root_motion_rotation], [method get_root_motion_scale], and [RootMotionView].
```

scotmcp replied:
Done and Squashed

AThousandShips reviewed (commented):
Please make your changes with `git commit --amend`

scotmcp committed:
Update  Animation.track_set_path and AnimationMixer.root_motion_track

Updated the usage description of [method Animation.track_set_path] and [member Animation.root_motion_track] to identify the parent is the root_node identified in animation player, and that the format is the same for each.

Remove extra symbols

akien-mga commented:
Thanks! And congrats for your first merged Godot contribution :tada:
